.github/workflows/collector.yml

```diff
@@ -86,32 +90,21 @@ jobs:
             -e @'${{ github.workspace }}/ansible/secrets.yml' \
             ansible/ci-build-collector.yml
 
-  build-collector-image-remote-vm:
```

Comment:
Please split the jobs back again. We used to have them unified, it is a nightmare to figure out what steps run on which conditions and having to add `if: something` to every single step is extremely tedious and annoying (specially when you forget one and weird errors start showing up).

Reply:
Done